pom.xml

```diff
@@ -157,29 +159,35 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
+      <version>${jackson.version}</version>
```

Comment:
No, it's not...

[fabric8-project-bom-with-platform-deps-2.2.169](https://repo1.maven.org/maven2/io/fabric8/fabric8-project-bom-with-platform-deps/2.2.169/fabric8-project-bom-with-platform-deps-2.2.169.pom)